openassessment/xblock/static/js/fixtures/templates.json

```diff
@@ -682,8 +684,52 @@
         "output": "oa_staff_area.html"
     },
     {
-        "template": "openassessmentblock/staff_area/student_info.html",
+        "template": "openassessmentblock/staff_area/oa_student_info.html",
```

Comment:
The way it works is that you can pass more than one set of parameters through the same template. The import distinction is the "output" which is the name of the resulting HTML for the template with the given context. This allows us to have many variations of student info for the different situations that we want to test.